src/spec/sync/utils/product.spec.coffee

```diff
@@ -1151,6 +1151,46 @@ describe 'ProductUtils', ->
         ]
       expect(update).toEqual expected_update
 
+    it 'should not modify original attributes', ->
```

Comment:
I am confused that original product is used as a base for attribute value and not new product